kustomize/README.md

```diff
@@ -1,107 +1,35 @@
-# Deploying Online Boutique variants with Kustomize
+# Deploying Online Boutique variations with Kustomize
 
-This page contains instructions on deploying variants of the [Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo) sample application using [Kustomize](https://kustomize.io/). Each variant is designed as a [**Kustomize Component**](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/components.md), so multiple variants can be composed together in the deployment.
+This page contains instructions on deploying variations of the [Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo) sample application using [Kustomize](https://kustomize.io/). Each variations is designed as a [**Kustomize component**](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/components.md), so multiple variations can be composed together in the deployment.
```

Comment:
Thought: I think we initially used "variants" because it's a [formal Kustomize glossary term](https://kubectl.docs.kubernetes.io/references/kustomize/glossary/#variant). But the more I think about it, "variation" sounds better and is more commonly used (according to Google search results count).

Reply:
Gotcha, yep, both were used, I went ahead and chose `variation` instead of `variant`. Either way, but just to be consistent ;)